crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -116,6 +116,17 @@ def repocheck
       end
     end
 
+    def ceph_healthy?
+      ceph_node = NodeObject.find("roles:ceph-mon AND ceph_config_environment:*").first
+      return true if ceph_node.nil?
+      ssh_retval = ceph_node.run_ssh_cmd("ceph health 2>&1")
```

Comment:
That's indeed possible, but could it exit with 0 while reporting anything else than HEALTH_OK?